test.sh

```diff
@@ -32,15 +35,20 @@ popd
 end eslint
 
 start cypress
+pushd hubmap
+python3 manage.py runserver &
+DJANGO_SERVER_PID=$!
```

Comment:
The server doesn't start instantly: you'll need to wait a bit for it to be available. You can write your own loop, or look at https://github.com/jeffbski/wait-on or https://linux.die.net/man/1/waitfor or https://github.com/vishnubob/wait-for-it

Reply:
thanks for sharing helpful, good references. Will take a look a this.

Reply:
since we no longer use cypress to test django routes (we already have tests in django to test the routes), this was autofixed since we no longer start django server here (that requires migrations).